pkgs/top-level/all-packages.nix

```diff
@@ -24751,11 +24751,9 @@ with pkgs;
     abseil-cpp = abseil-cpp_202103;
   };
 
-  protobuf3_24 = callPackage ../development/libraries/protobuf/3.24.nix { };
-  protobuf3_23 = callPackage ../development/libraries/protobuf/3.23.nix { };
-  protobuf3_21 = callPackage ../development/libraries/protobuf/3.21.nix {
-    abseil-cpp = abseil-cpp_202103;
-  };
+  protobuf3_24 = protobuf_24;
+  protobuf3_23 = protobuf_23;
+  protobuf3_21 = protobuf_21;
```

Comment:
These should be defined in `pkgs/top-level/aliases.nix`, and the packages that pin protobuf to a specific version should use the new attributes.

Reply:
According to the docs this is where they belong: https://github.com/NixOS/nixpkgs/tree/master/pkgs#overview.